addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -1496,7 +1496,7 @@ export class OdooEditor extends EventTarget {
         }
         if (joinWith) {
             const el = closestElement(joinWith);
-            fillEmpty(el);
+            el && fillEmpty(el);
```

Comment:
I added this because I reviewed all calls to `closestElement` and so far it seemed like the only one where I thought a check might be missing (most of the others are on `Selection.anchorNode`, `Selection.offsetNode`, `Range.startContainer` or `Range.endContainer` and I'm kind of assuming those can't be anything other than `TextNode` or `Element` (since I don't see how you could possibly have a cursor on the others).
In forward ports though we'll come across other cases. For 16.0, @rdeodoo mentioned something in `Sanitizer`, and I know of one related to table selections.

Reply:
It makes sense that `closestElement` might return `undefined` indeed.